joblib/test/test_parallel.py

```diff
@@ -286,17 +284,15 @@ def g(x):
 def test_parallel_timeout_success():
     # Check that timeout isn't thrown when function is fast enough
     for backend in ['multiprocessing', 'threading']:
-        nose.tools.assert_equal(
-            10,
-            len(Parallel(n_jobs=2, backend=backend, timeout=10)
-                (delayed(sleep)(0.001) for x in range(10))))
+        assert_equal(10, len(Parallel(n_jobs=2, backend=backend, timeout=10)
+                             (delayed(sleep)(0.001) for x in range(10))))
 
 
 @with_multiprocessing
 def test_parallel_timeout_fail():
     # Check that timeout properly fails when function is too slow
     for backend in ['multiprocessing', 'threading']:
-        nose.tools.assert_raises(
+        assert_raises(
```

Comment:
For readability, I would move TimeuotError on the same line and update the other lines alignment.